src/Language/PureScript/Ide/Prim.hs

```diff
@@ -11,10 +11,10 @@ idePrimDeclarations :: [IdeDeclarationAnn]
 idePrimDeclarations =
   primTypes <> primKinds <> primClasses
   where
-    primTypes = foreach (Map.toList PEnv.primTypes) $ \(tn, (kind, _)) ->
+    primTypes = foreach (Map.toList PEnv.allPrimTypes) $ \(tn, (kind, _)) ->
       IdeDeclarationAnn emptyAnn (IdeDeclType (IdeType (P.disqualify tn) kind []))
     primKinds = foreach (Set.toList PEnv.primKinds) $ \kn ->
       IdeDeclarationAnn emptyAnn (IdeDeclKind (P.disqualify kn))
-    primClasses = foreach (Map.toList PEnv.primClasses) $ \(cn, _) ->
+    primClasses = foreach (Map.toList PEnv.allPrimClasses) $ \(cn, _) ->
```

Comment:
@kRITZCREEK does this look right to you?

Reply:
I think the code inserts all of these declarations under the `Prim` module, if these declarations are now spread across different modules we'll need to account for that.

For example, in the current code:

```haskell
findType search filters currentModule = do
  modules <- Map.toList <$> getAllModules currentModule
  let insertPrim = (:) (C.Prim, idePrimDeclarations)
  pure (CompletionResult (getExactCompletions search filters (insertPrim modules)))
```

Reply:
Ok, neat, I can handle it from there. Thanks!